### PR TITLE
chore: add group select for analytics component

### PR DESCRIPTION
**Pull request by chenhaiyun**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

## Summary

(describe what this merge request does)

## Implementation highlights

(describe how the merge request does for feature changes, share the RFC link if it has)

## Test checklist

- [ ] add new test cases
- [x] all code changes are covered by unit tests
- [ ] end-to-end tests
  - [ ] deploy web console with CloudFront + S3 + API gateway
  - [ ] deploy web console within VPC
  - [ ] deploy ingestion server
    - [ ] with MSK sink
    - [ ] with KDS sink
    - [ ] with S3 sink
  - [ ] deploy data processing
  - [ ] deploy data modeling
    - [ ] new Redshift Serverless
    - [ ] provisioned Redshift
    - [ ] Athena
  - [ ] deploy with reporting

## Is it a breaking change

- [ ] add parameters without default value in stack
- [ ] introduce new service permission in stack
- [ ] introduce new top level stack module

## Miscellaneous

- [ ] introduce new symbol link source file(s) to be shared among infra code, web console frontend, and web console backend

### CODE REVIEW

File: frontend/src/pages/analytics/analytics-utils.ts [hunk 1]
```diff
@@ -170,6 +170,45 @@ export const parametersConvertToCategoryItemType = (
...
+  parameterItems: IMetadataEventParameter[]
+) => {
+  const computeMethodOptions: IAnalyticsItem[] = [
+    {
+      value: ExploreComputeMethod.EVENT_CNT,
+      label: i18n.t('analytics:options.eventNumber') ?? 'Event number',
+    },
+    {
+      label: defaultStr(i18n.t('analytics:sumGroup')),
+      value: 'SUM',
```
Comment (chenhaiyun) [inline frontend/src/pages/analytics/analytics-utils.ts:184]: set 'SUM' as enum. like `ExploreComputeMethod.SUM`

File: frontend/src/pages/analytics/analytics-utils.ts [hunk 2]
```diff
@@ -170,6 +170,45 @@ export const parametersConvertToCategoryItemType = (
...
+    ...numberAttributes,
+  ];
+  for (const parameter of numberParametersAndAttributes) {
+    computeMethodOptions[1].subList?.push({
+      value: parameter.name,
+      label: parameter.displayName,
+      name: parameter.name,
+      valueType: parameter.valueType,
+      category: parameter.category,
+      groupName: 'SUM',
```
Comment (chenhaiyun) [inline frontend/src/pages/analytics/analytics-utils.ts:205]: set 'SUM' as enum. like `ExploreComputeMethod.SUM`

File: frontend/src/components/eventselect/EventItem.tsx [hunk 3]
```diff
@@ -136,15 +143,54 @@ const EventItem: React.FC<EventItemProps> = (props: EventItemProps) => {
...
+              {showGroupSelectDropdown && (
+                <GroupSelectContainer
+                  categories={calcMethodOptions ?? defaultComputeMethodOptions}
+                  selectedItem={calcMethodOption ?? null}
+                  changeSelectItem={(item) => {
+                    if (item) {
+                      const newItem: any = { ...item };
+                      if (
+                        item.itemType === 'children' &&
+                        item.groupName === 'SUM'
```
Comment (chenhaiyun) [inline frontend/src/components/eventselect/EventItem.tsx:171]: set 'SUM' as enum. like `ExploreComputeMethod.SUM`

File: frontend/src/components/eventselect/EventItem.tsx [hunk 4]
```diff
@@ -136,15 +143,54 @@ const EventItem: React.FC<EventItemProps> = (props: EventItemProps) => {
...
+                      ) {
+                        newItem.label = t('analytics:sumGroupLabel', {
+                          label: item.label,
+                        });
+                      }
+                      if (
+                        item.itemType === 'children' &&
+                        item.groupName === 'AVG'
```
Comment (chenhaiyun) [inline frontend/src/components/eventselect/EventItem.tsx:179]: set 'AVG' as enum. like `ExploreComputeMethod.AVG`

File: src/control-plane/backend/lambda/api/service/attribution.ts [hunk 5]
```diff
@@ -110,8 +110,8 @@ export class AttributionAnalysisService {
       projectedColumns: [
         'Trigger Count',
         'Touch Point Name',
-        'Number of Attributed Triggers',
-        'Contribution',
+        'Number of Triggers with Conversion',
```
Comment (zxkane) [inline src/control-plane/backend/lambda/api/service/attribution.ts:113]: Where is the column name from? Should we make them constants?

Comment (tyyzqmf) [reply src/control-plane/backend/lambda/api/service/attribution.ts:113]: Only show in table when explore result.

**Review by zxkane — APPROVED** [review]

**Pull request merged**